src/components/ContributorsList/ContributorsList.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import List from '@mui/material/List';
```

Comment:
Small nit/question - it looks like these imports can be shortened to:
`import { List, Grid, Typography } from '@mui/material';`

It seems the material-ui docs tend to use individual import paths (like you have currently) in their examples. On the other hand, importing from `'@mui/material'` is concise. 

Any preference/thoughts?

Reply:
That's a good point! The reason for that is documented [here](https://mui.com/material-ui/guides/minimizing-bundle-size/)!

TL;DR - default direct imports are faster in the development environment and provide a smaller bundle size - though the latter will not matter in our case when we use a bundler that supports tree-shaking